css/friday.css

```diff
@@ -1,74 +1,82 @@
 @import "base.css";
 
 #friday {
-    font-family: futura, sans-serif;
-    text-align: center;
-    height: 100%;
-    margin-top: 10%;
-    width: 100%;
-    font-size: 12em;
+  font-family: futura, sans-serif;
+  text-align: center;
+  height: 100%;
+  margin-top: 10%;
+  width: 100%;
+  font-size: 12em;
 }
 
 .spring span {
-    background: url("../img/sakura.png");
+  background: url("../img/sakura.png");
 }
 
 .summer span {
-    background: url("../img/summer-leaf.png")
+  background: url("../img/summer-leaf.png")
+}
+
```

Comment:
We are going to do something kinda neat. Instead of relying on JS to transform our BG color. We are instead going to rely on CSS to change the color based on a CSS class. However, JS will set the class of our BG. “Now You're Thinking With Portals!”